cluster/minikube.go

```diff
@@ -70,6 +75,29 @@ func (cluster *minikubeCluster) Delete() error {
 	return nil
 }
 
+// DockerEnv returns the environment variables necessary to connect to the private docker repository in the kubernetes cluster
+func (cluster *minikubeCluster) DockerEnv() ([]string, error) {
+	statusCmd := exec.Command("minikube", "docker-env", "-p", cluster.Name)
+	stdout := &bytes.Buffer{}
+	statusCmd.Stdout = stdout
+	if err := statusCmd.Run(); err != nil {
+		return nil, err
+	}
+	if !statusCmd.ProcessState.Success() {
+		return nil, fmt.Errorf("Error creating cluster")
```

Comment:
error getting env? btw errors should start with lowercase

https://github.com/golang/go/wiki/Errors